CMakeLists.txt

```diff
@@ -103,6 +103,8 @@ target_link_libraries(${PROJECT_NAME} GothicShaders)
 
 if(NOT MSVC)
   target_compile_options(${PROJECT_NAME} PRIVATE -Wall -Wconversion -Wno-strict-aliasing -Werror)
+  # -rdynamic is needed to allow obtaining meaningful backtraces
+  target_link_options(${PROJECT_NAME} PRIVATE -rdynamic)
```

Comment:
`if(UNIX)` ?
-rdynamic seems not working in mingw

Reply:
It's good there's a CI attached :-) Added UNIX check, should be much more specific.